src/token/token.service.ts

```diff
@@ -89,6 +90,25 @@ export class TokenService {
     const sessionDTO = userSession.session;
     sessionDTO.tokenExpiration = tokenExpiration;
 
+    // Bypass logic
+    if (
+      this.configService.get<string>('app.env') === 'development' &&
```

Comment:
change logic to not equal to 'production' so that this can be enabled for all non-prod environments if needed